packages/appstore/src/modules/wallets/wallets.scss

```diff
@@ -1,6 +1,6 @@
 .wallets-module {
     display: flex;
-    min-height: 92.8vh;
+    min-height: calc(100vh - 84px);
```

Comment:
why `84px` 🤔

Reply:
hey @aizad-deriv , that how we calculating our viewport for desktop. 84 px is a sum of header and footer in that case